.github/pull_request_template.md

```diff
@@ -1,8 +1,11 @@
 ## Description
 
-Please edit this to include a summary of the change (what and why).
+Please edit this to include a summary of the change (what).
 Include screenshots if you modify the UI.
 
+## Motivation
+Please edit this to include the reason why we are making this change.
```

Comment:
If the Linear issue already describes the motivation then do we need to repeat it in the PR description?

Reply:
When the issue already describes the motivation, I think this section isn't needed in those circumstances.

Reply:
This is an optional reminder. We suggested this in the Frontend Guild mostly because of the UI refactors that don't provide any context and don't have linear tickets (many linear tickets have not also provided any context besides the description).